tests/e2e/pageobjects/git-providers/OauthPage.ts

```diff
@@ -17,140 +17,186 @@ import { TimeoutConstants } from '../../constants/TimeoutConstants';
 
 @injectable()
 export class OauthPage {
-    private readonly loginForm: By;
```

Comment:
On our last QE sync call we had an agreement to have all locators as `private static readonly` and By type. It will be very nice if you will fix it in this pageobject class you currently working on ☺️

Reply:
https://docs.google.com/document/d/1_BlAqY1s8Ou__9iMQR2ZtmUnkuJ1VHDVtSBTV5-_Uv0/edit#heading=h.yr66jvhjgpfu